src/modules/sub-items/components/table-view/table.view.component.js

```diff
@@ -51,8 +62,15 @@ export default class TableViewComponent extends Component {
 
     getColumnsVisibilityFromLocalStorage() {
         const columnsVisibilityData = localStorage.getItem(COLUMNS_VISIBILITY_LOCAL_STORAGE_DATA_KEY);
+        const columnsVisibility = { ...DEFAULT_COLUMNS_VISIBILITY };
 
-        return columnsVisibilityData ? JSON.parse(columnsVisibilityData) : DEFAULT_COLUMNS_VISIBILITY;
+        Object.entries(JSON.parse(columnsVisibilityData)).forEach(([id, isVisible]) => {
```

Comment:
I think if `columnsVisibilityData` will not be set in the localStorage this will throw an exception.

Reply:
Good catch!